src/rollback_to_stable/rts_api.c

```diff
@@ -8,6 +8,24 @@
 
 #include "wt_internal.h"
 
+/*
+ * __rts_assert_timestamps_unchanged --
+ *     Wrapper for some diagnostic assertions related to timestamps.
```

Comment:
how about changing it from `timestamps` to `global timestamps`?

Reply:
Yep fair enough. Done in 8f205f6.